File: sylk-iot/iot-server/subscriber.js
const stub = require('./clients/typescript');

const iot = new stub.IotServiceV1();

iot.Subscribe({
    name: 'sine'
}).subscribe(evt => {
    console.log('sine',evt)
})

iot.Subscribe({
    name: 'cosine'
}).subscribe(evt => {
    console.log('cosine',evt)
})

File: sylk-iot/iot-server/subscriber.py
from clients.python import IotService_v1
from clients.python.protos.sylklabs.iot.v1.iot_pb2 import Topic

stub = IotService_v1(client_opt={
    'port': 50001
})
events = stub.Subscribe(Topic(
    name='default'
))

for evt in events:
    print(evt)